crates/optimism/rpc/src/eth/mod.rs

```diff
@@ -56,10 +58,10 @@ pub type EthApiNodeBackend<N> = EthApiInner<
 ///
 /// This type implements the [`FullEthApi`](reth_rpc_eth_api::helpers::FullEthApi) by implemented
 /// all the `Eth` helper traits and prerequisite traits.
-#[derive(Clone, Deref)]
+#[derive(Clone, Deref, DerefMut)]
 pub struct OpEthApi<N: FullNodeComponents> {
     #[deref]
-    inner: Arc<EthApiNodeBackend<N>>,
+    inner: Arc<(EthApiNodeBackend<N>, Option<SequencerClient>)>,
```

Comment:
```suggestion
    inner: Arc<EthApiNodeBackend<N>>,
    sequencer: Option<SequencerClient>,
```
better to ref fields by name rather than by index